CHANGES.md

```diff
@@ -1,3 +1,206 @@
+## stable-2.4.0
+
+This release adds traffic splitting functionality, support for the Kubernetes
+Service Mesh Interface (SMI), graduates high-availability support out of
+experimental status, and adds a tremendous list of other improvements,
+performance enhancements, and bug fixes
+
+Linkerd's new traffic splitting feature allows users to dynamically control the
+percentage of traffic destined for a service. This powerful feature can be used
+to implement rollout strategies like canary releases and blue-green deploys.
+
+Support for the [Service Mesh Interface](https://smi-spec.io) (SMI) makes it
```

Comment:
I would just merge the first sentence of this paragraph into the previous one, and kill the Flagger reference -- doesn't feel right for release notes.